### PR TITLE
Add testimonials card

### LINKED ISSUE #5: Reviews Card
- [ ] this should still use manrope. use closest matching style for now

Image

### PR DESCRIPTION
**Pull request by mikabusante**
closes #5

### CODE REVIEW

File: data/testimonials.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
+[
```
Comment (mikabusante) [inline data/testimonials.json:1]: this data was originally in a component, but I moved it into a JSON file. Maybe makes it easier to find for future editing?

File: styles/globals.scss [hunk 2]
```diff
@@ -150,8 +152,12 @@ $xxl: 1400px;
...
-  
+
+  .full-width {
+    @apply ml-[50%] w-screen -translate-x-1/2;
+  }
+
   .full-width-bg {
-    @apply top-0 bottom-0 absolute left-1/2 -translate-x-1/2 w-[100vw] -z-10;
+    @apply absolute top-0 bottom-0 left-1/2 -z-10 w-[100vw] -translate-x-1/2;
   }
```
Comment (mikabusante) [inline styles/globals.scss:162]: could we consolidate these classes? i think they're the same except for the absolute positioning. maybe we could move `absolute top-0 bottom-0` to the footer?

File: tailwind.config.js [hunk 3]
```diff
@@ -52,34 +52,36 @@ module.exports = {
...
+      "main-blurple": "var(--main-blurple)",
+      "accent-blurple": "var(--accent-blurple)",
+      "blurple-text-on-black": "var(--blurple-text-on-black)",
+      lime: "var(--lime)",
+      goldenrod: "var(--goldenrod)",
+      "gray-1": "var(--gray-1)",
+      "gray-2": "var(--gray-2)",
+      "gray-3": "var(--gray-3)",
+      "gray-4": "var(--gray-4)",
+      "gray-5": "var(--gray-5)",
```
Comment (mikabusante) [inline tailwind.config.js:69]: moved these colors to css variables so we're not writing the hex codes twice. Alternatively, we could just define the colors in the tailwind config but it's kinda nice to be able to reference these from the dev tools :~)

File: pages/index.js [hunk 4]
```diff
@@ -44,6 +53,27 @@ function HomeHero() {
...
+  return (
+    <section className="full-width bg-gray-5 pt-20 pb-28">
+      <h2 class="h3 pb-16 text-center">
+        <FormattedMessage
+          id="home.testimonials.title"
+          defaultMessage="What our users are saying"
+        />
+      </h2>
+      <div class="m-auto grid max-w-site gap-gutter px-6 md:grid-cols-3 lg:px-16">
+        {/* TODO(mika): these cards should be rendered in a carousel*/}
```
[the remark this answers is not in the document]
Comment (danielgamage) [reply pages/index.js:66]: i think plain flickity sounds ok—there's stuff like https://github.com/yaodingyd/react-flickity-component (supposedly doesn't support SSR / have typescript types), and a lot of the other react carousel demos feel not great or are 100kb. open to other options though and don't feel like it needs to get done in this PR

File: components/LinkButton.jsx [hunk 5]
```diff
@@ -2,18 +2,27 @@ import Link from "next/link"
...
-            "w-full": fullWidth,
-            "bg-accent-blurple text-white": !light,
-            "bg-white text-accent-blurple hover:text-white": light,
-          }
+          "b3 block rounded border-2 p-4 text-center !font-600 transition-all hover:border-dark-blurple hover:bg-dark-blurple",
+          borderless ? "border-white" : "border-accent-blurple",
+          fullWidth ? "w-full" : "w-max",
+          light
+            ? "bg-white text-accent-blurple hover:text-white"
+            : "bg-accent-blurple text-white"
```
[the remark this answers is not in the document]
Comment (mikabusante) [reply components/LinkButton.jsx:22]: ya before this i had been marking the override styles as  `!important` but didn't seem right and it wasn't very readable

File: pages/index.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (mikabusante) [reply pages/index.js:66]: ya agree. i'll look into other options and let's come back to it later.

**Pull request merged**